### PR TITLE
Skbio removal

### PR DESCRIPTION
**Pull request by wwood**

### CODE REVIEW

File: graftm/dendropy_tree_cleaner.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from dendropy import Tree
+import copy
```
Comment (geronimp) [inline graftm/dendropy_tree_cleaner.py]: needed?

File: graftm/dendropy_tree_cleaner.py [hunk 2]
```diff
@@ -19,3 +19,36 @@ def write_fasttree_newick(self, tree, output_file_io):
                                 suppress_internal_taxon_labels=True,
                                 preserve_spaces=False) # Labels must not be quoted.
         output_file_io.write(newick.replace("'",'').replace(' ','_'))
+            
+    def match_alignment_and_tree_sequence_ids(self, sequence_names, tree):
+        '''Check to make sure that the sequences specified in the alignment
+        and the tree are the same, otherwise raise an Exception detailing
+        the problem for the user to fix
+
+        Params
```
Comment (geronimp) [inline graftm/dendropy_tree_cleaner.py]: not important, but the rest of the code calls this "Parameters"

**Comment by geronimp** [comment]
This all seems pretty straight forward to me

**Comment by wwood** [comment]
Thanks for keeping me honest, I made your suggested changes and pushed.

**Pull request closed without merging**